src/main/java/org/broadinstitute/dsm/util/externalShipper/ExternalShipper.java

```diff
@@ -12,12 +12,14 @@
 
     public void orderKitRequests(ArrayList<KitRequest> kitRequests, EasyPostUtil easyPostUtil, KitRequestSettings kitRequestSettings, String shippingCarrier) throws Exception;
 
-    public void orderStatus(ArrayList<KitRequest> kitRequests) throws Exception;
+    /**
+     * Checks for incomplete orders and updates the internal status
+     * of the order
+     */
+    void updateOrderStatusForPendingKitRequests(int instanceId);
 
     public void orderConfirmation(long startDate, long endDate) throws Exception;
 
     public void orderCancellation(ArrayList<KitRequest> kitRequests) throws Exception;
 
-    public  ArrayList<KitRequest> getKitRequestsNotDone(int instanceId);
```

Comment:
I moved this to `GBFRequestUtil`, rather than the interface, since it's now used only when testing `GBFRequestUtil`.